app/models/episode.rb

```diff
@@ -258,33 +258,20 @@ def media_files
     end
   end
 
+  # API updates for media= ... just append new files and reprocess
   def media_files=(files)
-    update_contents(files)
-  end
-
-  def update_contents(files)
```

Comment:
This method wasn't actually called anywhere apart from `media_files =`, through the HalApi/Roar representer stuff.  So I rewrote it to just _always_ append new Contents, even if the url hasn't changed.  (It'll do the usual thing where it waits for the fixer callback to swap the files, though).

The `EpisodeEntryHandler` and `EpisodeStoryHandler` have their own "check if the files changed" logic built into them, but they _do_ rely on the `find_existing_content` method, which I've left intact.